app/views/works/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_for @work do |f| %>
+  <%= f.label :Category %>
+  <%= f.select :category, options_for_select(["album", "book", "movie"]) %>
```

Comment:
You can change this so that it defaults to the current value of category.  with `selected: @work.category`